web/src/components/task/task-sidebar/finish-button/confirm-modal/comfirm-modal.module.scss

```diff
@@ -0,0 +1,3 @@
+.modalBody {
```

Comment:
I propose to use kebab case for naming just to use more usual way to define classes.

Reply:
fixed